code/modules/vehicles/mecha/mech_fabricator.dm

```diff
@@ -149,7 +149,7 @@
 /obj/machinery/mecha_part_fabricator/emag_act(mob/user, obj/item/card/emag/emag_card)
 	if(obj_flags & EMAGGED)
 		return FALSE
-	if(user.job != JOB_ROBOTICIST)
+	if(!(user.job == JOB_ROBOTICIST || user.job == JOB_RESEARCH_DIRECTOR))
```

Comment:
Why does this even have a job lock on it anyways? No other emag act has a job lock like this.

Reply:
it was requested by oranges

Reply:
Where did he request this?

Reply:
In either the discord or the absolute mess that is the PR that added the Justice mech.